content/blog/custom-docker-images-for-emr-serverless.md

```diff
@@ -0,0 +1,199 @@
+
+<!--StartFragment-->
+
+aws — region <region> emr-serverless create-application \
+— release-label emr-6.9.0 \
+— type “SPARK” \
+— name emr-application-1 \
+**— image-configuration ‘{ “imageUri”: “<your AWS account ID>.dkr.ecr.<region>.[amazonaws.com/<ecr_registry_name:](http://amazonaws.com/emr-serverless-ci-examples:emr-serverless-ci-ml)image_name>” }’**
+
+<!--EndFragment-->
+
+**Note:** Image URI can be copied from AWS ECR registry.
+
+### **4b) Via EMR Serverless Settings in the EMR Management console**
+
+##### **EMRServerless Custom Image Settings section in AWS EMR Serverless management console:**
+
+![EMR Serverless Management Console Docker section](/img/customimagesettings-in-emrserverless.png)
+
+##### Browse and select custom EMRServerless image from AWS ECR private registry in the same region:
+
+![EMR Serverless Management Console to choose the Docker Image](/img/selectionofimagefromecr_registry.png)
+
+### **Reference links to AWS documentations:**
+
+* *Steps to build custom docker image for EMRServerless:* <https://aws.amazon.com/blogs/big-data/add-your-own-libraries-and-application-dependencies-to-spark-and-hive-on-amazon-emr-serverless-with-custom-images/>
+* *How to pull docker image from AWS ECR registry:* <https://docs.aws.amazon.com/AmazonECR/latest/userguide/docker-pull-ecr-image.html>
+* *How to authorize AWS ECR registry with docker client:* <https://docs.aws.amazon.com/AmazonECR/latest/userguide/registry_auth.html>
+* *How to push docker image to AWS ECS registry:* <https://docs.aws.amazon.com/AmazonECR/latest/userguide/docker-push-ecr-image.html>
+* *AWS EMRServerless CLI reference:* <https://docs.aws.amazon.com/cli/latest/reference/emr-serverless/index.html>
+* *How to customize image for multi CPU architecture:* <https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/docker-custom-images-steps.html>
+
+<!--EndFragment-->
+
+## **About the Authors:**
+
+![](/img/niranjan_2.jpg)
+
+<!--StartFragment-->
+
+D R Niranjan is a Senior software/Cloud application developer at HPE. Experienced in developing software and cloud applications for HPE Servers and Storage systems. Technically skilled in Python, Java, Scala, Spark programming, AWS, Dockers and Kubernetes, PgSQL/RDBMS. You can have a look at his LinkedIn profile here: <https://www.linkedin.com/in/niranjan-d-r/>
+
+<!--EndFragment-->
+
+![](/img/sagar-pic_2.jpg "sagar-nyamagouda@hpe.com")
+
+Sagar Nyamagouda holds a  B.E(Information Science and Engineering) from BMS College of Engineering (BMSCE), Bengaluru and M.Tech in Software Systems from BITS Pilani. He is an experienced R&D Engineer working on Big Data Technologies and building AI/ML pipelines for give real time insights to customers. An AI/ML enthusiast. Currently working with HPE enabling advanced insights for DSCC - an HPE Flagship cloud offering.   LinkedIn Profile: www.linkedin.com/in/sagarny
```

Comment:
In the third line, change "for give real time" to "to give real time". I would combine the last couple of sentences to read:
"An AI/ML enthusiast, he is currently working with HPE enabling advanced insights for DSCC - an HPE flagship cloud offering." If we really want to follow Brand requirements specifically, "DSCC" needs to be spelled out, and somehow relate it to the HPE GreenLake Cloud Platform.